lms/djangoapps/grades/new/course_data.py

```diff
@@ -0,0 +1,89 @@
+from lms.djangoapps.course_blocks.api import get_course_blocks
+from xmodule.modulestore.django import modulestore
+from ..transformer import GradesTransformer
+
+
+class CourseData(object):
```

Comment:
I'm wondering whether this class might be useful in a non-grades-specific context. Would there be value in placing it in a more general-use location?